tests/all/test_passes.rs

```diff
@@ -142,18 +165,20 @@ fn test_pass_manager_builder() {
 
     pass_manager_builder.populate_module_pass_manager(&module_pass_manager);
 
-    let module2 = module.clone();
-
     #[cfg(any(feature = "llvm4-0", feature = "llvm5-0"))]
     assert!(!module_pass_manager.run_on(&module));
     #[cfg(not(any(feature = "llvm4-0", feature = "llvm5-0")))]
     assert!(module_pass_manager.run_on(&module));
 
-    let lto_pass_manager = PassManager::create(());
+    #[cfg(not(feature = "llvm15-0"))]
```

Comment:
Why exclude 15?

Reply:
Oh - you mentioned the pass manager got removed in 15?

Reply:
Yes, the LTO was removed from the old pass manager.